pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/FieldDeclarationsShouldBeAtStartOfClass.xml

```diff
@@ -64,10 +65,11 @@ public class MyTest extends AbstractJUnit4SpringContextTests {
 
     <test-code>
         <description>#1126 False positive with FieldDeclarationsShouldBeAtStartOfClass for static enums</description>
-        <expected-problems>0</expected-problems>
+        <expected-problems>3</expected-problems>
```

Comment:
that looks suspicious... did the default value of "ignoreEnumDeclarations" change? Hm... no, but this property doesn't seem to work anymore like before....

Reply:
The enum is followed by a semicolon, and the EmptyDeclaration node was not whitelisted